torch/onnx/symbolic.py

```diff
@@ -619,3 +619,15 @@ def symbolic(g, input, all_weights, h0, **fkwargs):
         return prev_output, h_outs
 
     return torch.onnx.symbolic_override(symbolic)
+
+
+def instance_norm_symbolic_builder(*args, **kwargs):
```

Comment:
you don't need this wrapper as args and kwargs are empty. Just do `symbolic_override`(symbolic) and that's it

Reply:
With #4799 just have here:

```
# not sure how to inherit the default values...
def symbolic(g, input, weight, bias, eps, **unused_rest_kwargs):
        input_type = input.type().scalarType()
        if not weight:
            weight = g.constant(1.0, [input.type().sizes()[1]], input_type)
        if not bias:
            bias = g.constant(0.0, [input.type().sizes()[1]], input_type)
        return g.op("InstanceNormalization", input, weight, bias, epsilon_f=eps)
```

And just add above:

```
@torch.onnx.symbolic_override_first_arg_based(torch.onnx.symbolic.instance_norm)
def instance_norm(....): ...
```